src/Build/BackEnd/Components/Logging/LoggingService.cs

```diff
@@ -462,6 +467,16 @@ public int MaxCPUCount
             set => _maxCPUCount = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value that will error when the build process fails an incremental check.
+        /// </summary>
+        public bool Question
```

Comment:
The name might appear unclear on the first sight - how about `IsQuestionSwitchUsed` or `IsQuestionMode`?

Reply:
But we already have the same naming as public property in BuildParameters - so probably let's rather keep as is to stay consistent.

Reply:
I am using the name from Make.  Another common name is dry-run.

Reply:
Keeping it as is.